package.json

```diff
@@ -14,33 +14,39 @@
   ],
   "license": "MIT",
   "bin": {
-    "uuid": "./bin/uuid"
+    "uuid": "dist/bin/uuid"
   },
   "sideEffects": false,
-  "main": "index.js",
-  "module": "esm-browser/index.js",
+  "main": "dist/index.js",
+  "module": "dist/esm-browser/index.js",
+  "files": [
+    "CHANGELOG.md",
+    "LICENSE.md",
+    "README.md",
+    "dist"
+  ],
   "devDependencies": {
-    "@babel/cli": "~7.6.4",
-    "@babel/core": "~7.6.4",
-    "@babel/preset-env": "~7.6.3",
-    "@commitlint/cli": "~8.2.0",
-    "@commitlint/config-conventional": "~8.2.0",
-    "babel-eslint": "~10.0.3",
-    "babel-plugin-add-module-exports": "~1.0.2",
-    "browserstack-local": "~1.4.2",
-    "eslint": "~6.4.0",
-    "eslint-config-prettier": "~6.4.0",
-    "eslint-plugin-prettier": "~3.1.1",
-    "esm": "~3.2.25",
-    "http-server": "~0.12.0",
-    "husky": "~3.0.9",
-    "jest": "~24.9.0",
-    "lint-staged": "~9.4.3",
-    "npm-run-all": "~4.1.5",
-    "prettier": "~1.18.2",
-    "runmd": "1.2.1",
-    "selenium-webdriver": "~3.6.0",
-    "standard-version": "7.0.0"
+    "@babel/cli": "^7.8.3",
```

Comment:
Change .npmrc to have `save-prefix="^"`

(For the record, I'd be okay removing version prefixes altogether so we have explicit versions throughout.)

Reply:
I switched to explicit versions.